docs/fieldnorm/risk_management.md

```diff
@@ -1,16 +1,35 @@
-The MANUFACTURER shall include or reference in the software development plan, a plan to conduct the ACTIVITIES and TASKS of the software RISK MANAGEMENT PROCESS, including the management of RISKS relating to SOUP. [Class A, B, C] 7.4 RISK MANAGEMENT of software changes
-7.4.1 Analyse changes to MEDICAL DEVICE SOFTWARE with respect to SAFETY
-The MANUFACTURER shall analyse changes to the MEDICAL DEVICE SOFTWARE (including SOUP) to determine whether:
-1. additional potential causes are introduced contributing to a hazardous situation; and
-2. additional software RISK CONTROL measures are required.
-[Class A, B, C]
+The bioinformatics risk assessment should integrate with the laboratory risk management procedure, which can use the existing method described in the laboratories QMS, such as SAFER (Safety Assurance Factors for EHR Resilience) or FMEA (Failure Modes and Effects Analysis). In the scenario where the bioinformatics team opts for an alternative approach, this alternative approach needs to be documented.
 
-HvD: note from lecture about new ISO:
-How does risk management contribute to patient safety?
-Do we need more trend analsys??
\ No newline at end of file
+Often, bioinformatics software is a part of the laboratory test, hence it should be clear which part of the laboratory test is covered by the bioinformatics risk management.  For instance, for sequencing the bioinformatics risk management starts with the initial sequencer output and ends with the storage of results.
+
+## Risk Management in Software Life Cycle Processes
+Risk management is part of the software life cycle process and is required for every (standalone) pipeline/workflow or software tool validated for diagnostics use. Risk management can be broken down into 5 steps, where step 2 and 3 should be part of the laboratories QMS. When the bioinformatics team opts for an alternative approach, step 2 and 3 need to be documented.  
+
+1. Risk Identification involves a comprehensive analysis of risk factors such as project scope, requirements, technology stack, resource availability, and external dependencies.
+Typically bioinformatics risks involve topics like:
+   - Software goal
+   - Software installation (and dependencies)
+   - Running software tools
+   - Interactions / connections with databases / other software / other hardware
+   - Data storage
+   - User errors
+   - Compute environment is down / not reachable
+   - Technical challenges
+   - Resource constraints
+2. Risk Assessment: Once risks are identified, they need to be evaluated to determine their potential impact and likelihood. This assessment allows prioritization, focusing on high-impact and high-probability risks. Risk assessment involves quantitative or qualitative analysis, aiding in the allocation of resources and attention to the most critical areas.
+3. Risk Mitigation and Planning: Based on the risks assessment, risks must be mitigated. 
```

Comment:
What are the criteria to consider a risk mitigated?
Can a risk also be diminished in impact and likelihood?
What if a risk can not possibly be mitigated?
Are there situations where projects come through despite having risks?
Are there situations where projects must be cancelled because of the risks?